cypress/integration/recipe_detail_spec.js

```diff
@@ -0,0 +1,354 @@
+    cy.get('[data-testid="6-ingredient-name-and-measure"]').contains('6 leaves');
+    cy.get('[data-testid="7-ingredient-name-and-measure"]').contains('Parmigiano-Reggiano');
+    cy.get('[data-testid="7-ingredient-name-and-measure"]').contains('spinkling');
+
+    cy.get('[data-testid="instructions"]').contains('Bring a large pot of water to a boil. Add kosher salt to the boiling water, then add the pasta. Cook according to the package instructions, about 9 minutes.\r\nIn a large skillet over medium-high heat, add the olive oil and heat until the oil starts to shimmer. Add the garlic and cook, stirring, until fragrant, 1 to 2 minutes. Add the chopped tomatoes, red chile flakes, Italian seasoning and salt and pepper to taste. Bring to a boil and cook for 5 minutes. Remove from the heat and add the chopped basil.\r\nDrain the pasta and add it to the sauce. Garnish with Parmigiano-Reggiano flakes and more basil and serve warm.');
+
+    cy.get('[data-testid="video"]').should('exist');
+
+    cy.get('[data-testid*="recomendation-card"]').should('exist');
+  });
+
+  it('verifica se os elementos existem na tela de detalhes de bebida', () => {
+    cy.visit('http://localhost:3000/bebidas/178319', {
+      onBeforeLoad(win) {
+        win.fetch = fetchMock;
+      },
+    });
+
+    cy.get('[data-testid="recipe-photo"]')
+      .should('have.prop', 'tagName')
+      .should('eq', 'IMG');
+
+    cy.get('[data-testid="recipe-title"]').contains('Aquamarine');
+
+    cy.get('[data-testid="recipe-category"]').contains('Cocktail');
+
+    cy.get('[data-testid="0-ingredient-name-and-measure"]').contains('Hpnotiq');
+    cy.get('[data-testid="0-ingredient-name-and-measure"]').contains('2 oz');
+    cy.get('[data-testid="1-ingredient-name-and-measure"]').contains('Pineapple Juice');
+    cy.get('[data-testid="1-ingredient-name-and-measure"]').contains('1 oz');
+    cy.get('[data-testid="2-ingredient-name-and-measure"]').contains('Banana Liqueur');
+    cy.get('[data-testid="2-ingredient-name-and-measure"]').contains('1 oz');
+
+    cy.get('[data-testid="instructions"]').contains('Shake well in a shaker with ice.\r\nStrain in a martini glass.');
+
+    cy.get('[data-testid*="recomendation-card"]').should('exist');
+  });
+});
+
+describe('As recomendações para receitas de comida deverão ser bebidas e vice-versa', () => {
+  it('verifica se a requisição para a API de bebidas foi realizada', () => {
+    cy.visit('http://localhost:3000/comidas/52771', {
+      onBeforeLoad(win) {
+        cy.spy(win, 'fetch');
+      },
+    });
+
+    cy.window()
+      .its('fetch')
+      .should('be.calledWithMatch', 'https://www.thecocktaildb.com/');
+  });
+
+  it('verifica se a requisição para a API de comidas foi realizada', () => {
+    cy.visit('http://localhost:3000/bebidas/178319', {
+      onBeforeLoad(win) {
+        cy.spy(win, 'fetch');
+      },
+    });
+
+    cy.window()
+      .its('fetch')
+      .should('be.calledWithMatch', 'https://www.themealdb.com/');
```

Comment:
Acredito que aqui você poderia ser mais específico, já que os 6 cards do requisito seguinte devem vir desse endpoint:

```suggestion
      .should('be.calledWith', 'https://www.themealdb.com/api/json/v1/1/search.php?s=');
```